docs/resources.html

```diff
@@ -80,10 +82,14 @@ <h2 id="open-science">Open science</h2>
 				<h3 id="resources-for-authors">Resources for authors</h3>
 				<p>Aczel et al. (2020) present a consensus-based <a href="https://www.nature.com/articles/s41562-019-0772-6" target="_blank">checklist</a> to improve and document the transparency of research reports in social and behavioral research along with an <a href="http://www.shinyapps.org/apps/TransparencyChecklist/" target="_blank">online application</a> that allows users to complete the checklist and generate a report that they can submit with their manuscript or post to a public repository.</p>
 				<h3 id="data-and-code">Data and code</h3>
-				<p> A set of stable core practices has begun to emerge with regard to data management and sharing. While all are readily available, some require more effort on the part of journals. For example, requiring authors to share data via a trusted digital repository (and not e.g. via a personal website) is easier than checking that the author's code runs. </p>
+				<p> A set of stable core practices has begun to emerge with regard to data management and sharing. While all are readily available, some require more effort on the part of journals. For example, requiring authors to share data via a trusted digital repository (and not e.g. via a personal website) is easier than checking that the author's code runs. Iain Hrynaszkiewicz has written a chapter outlining “<a href="https://doi.org/10.1007/164_2019_290" target="_blank">Publishers’ Responsibilities in Promoting Data Quality and Reproducibility</a>” that describes practical approaches being taken by publishers to promote rigor and transparency in data practices.</p>
```

Comment:
Is Iain's affiliation relevant here? I'd be inclined to include it in parentheses